dockstore-integration-testing/src/test/java/io/dockstore/webservice/ServiceIT.java

```diff
@@ -217,7 +218,14 @@ public void testGitHubAppEndpoints() throws Exception {
         // Test user endpoints
         UsersApi usersApi = new UsersApi(webClient);
         services = usersApi.userServices(service.getUsers().get(0).getId());
-        List<io.swagger.client.model.Workflow> workflows = usersApi.userWorkflows(service.getUsers().get(0).getId());
+        List<io.swagger.client.model.Workflow> workflows = new ArrayList<>();
+        try {
+            workflows.addAll(usersApi.userWorkflows(service.getUsers().get(0).getId()));
```

Comment:
Not really sure I understand what is going on here. 
I would have expected services on line 220 to be filled with services, then workflows here to be filled with workflows. 
It seems odd to me to expect a 500 error?